src/scp/SCPDriver.cpp

```diff
@@ -80,13 +79,14 @@ static const uint32 hash_K = 3;
 
 static uint64
 hashHelper(uint64 slotIndex, Value const& prev,
-           std::function<void(SHA256&)> extra)
+           std::function<void(std::vector<xdr::opaque_vec<>>&)> extra,
+           SCPDriver const& driver)
```

Comment:
as you're in the SCPDriver class, you can simply define this helper function as a private method

Reply:
Thanks. In retrospect that should have been obvious. :)